tests/integration/test_vulnerability_detector/test_feeds/test_invalid_values_redhat_feeds.py

```diff
@@ -0,0 +1,151 @@
+ids = ['REDHAT_configuration']
+
+# Read JSON data template
+with open(vulnerabilities_data_path, 'r') as f:
+    nvd_vulnerabilities = json.loads(f.read())
+
+system_data = {"target": "RHEL8", "os_name": "CentOS Linux", "os_major": "8", "os_minor": "1", "name": "centos8"}
+
+expected_error_message = r"ERROR: \(\d+\): CVE database could not be updated."
+expected_success_message = r"INFO: \(\d+\): The update of the 'Red Hat Enterprise Linux' feed finished successfully."
+
+# Custom inputs to check
+inputs = [None, "", "dummy value", 12345, ['1', '2', '3', '4', '5'], "ñ", "テスト", "ИСПЫТАНИЕ", "测试", "اختبار"]
+str_type = "<class 'str'>"
+list_type = "<class 'list'>"
+
+# Redhat fields to check
+field_data = [
+    {"field": "CVE", "type": str_type},
+    {"field": "severity", "input": "", "type": str_type},
+    {"field": "public_date", "type": str_type},
+    {"field": "advisories", "type": list_type},
+    {"field": "bugzilla", "type": str_type},
+    {"field": "bugzilla_description", "type": str_type},
+    {"field": "cvss_score", "type": str_type},
+    {"field": "cvss_scoring_vector", "type": str_type},
+    {"field": "CWE", "type": str_type},
+    {"field": "affected_packages", "type": list_type},
+    {"field": "resource_url", "type": str_type},
+    {"field": "cvss3_scoring_vector", "type": str_type},
+    {"field": "cvss3_score", "type": str_type}
+]
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+
+
+# Fixtures
+@pytest.fixture(scope='module', params=configurations, ids=ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture()
+def modify_feed(field_info, custom_input, request):
+    """
+    Modify the redhat OVAL feeds, setting a test field value
+    """
+    backup_data = json.loads(read_file(custom_redhat_oval_feed_path))[0]
+
+    modified_data = dict(backup_data)
+
+    modified_data[field_info['field']] = custom_input
+
+    write_file(custom_redhat_oval_feed_path, json.dumps([modified_data], indent=4, ensure_ascii=False))
+
+    clean_vuln_and_sys_programs_tables()
+
+    control_service('restart', daemon='wazuh-modulesd')
+
+    set_custom_system(os_name=system_data['os_name'], os_major=system_data['os_major'],
+                      os_minor=system_data['os_minor'], name=system_data['name'])
+
+    yield
+
+    write_file(custom_redhat_oval_feed_path, json.dumps([backup_data], indent=4))
+
+    clean_vuln_and_sys_programs_tables()
+
+    truncate_file(LOG_FILE_PATH)
+
+
+def check_feed_imported_successfully():
+    """
+    Check that redhat OVAL feeds have been imported successfully
+    """
+    check_vulnerability_event(
+        wazuh_log_monitor=wazuh_log_monitor, update_position=False, timeout=10,
+        callback=expected_success_message,
+        error_message=f"Could not find the message: 'Red Hat Enterprise Linux' feed finished successfully"
```

Comment:
This function can be used to check if other feeds have been successfully imported, so it would be great if the feed's name can be set by a variable.

Reply:
Done in ab1c229f9625689d173018ae41d5437228ded6be